### PR TITLE
Add check types and bump to 0.0.2

### PR DESCRIPTION
**Pull request by acraileanu**
Add CheckType constants based on the API reference:
https://passfort.com/developer/v4/data-reference/CheckType

Bumped version to 0.0.2

### CODE REVIEW

File: lib/passfort.rb [hunk 1]
```diff
@@ -8,6 +8,21 @@
 require "passfort/client"
 
 module Passfort
+  module CheckType
+    IDENTITY_CHECK = "IDENTITY_CHECK"
+    DOCUMENT_FETCH = "DOCUMENT_VERIFICATION"
```
Comment (hmac) [inline lib/passfort.rb]: should these be the same?

File: lib/passfort.rb [hunk 2]
```diff
@@ -8,6 +8,21 @@
...
+    DOCUMENT_FETCH = "DOCUMENT_FETCH"
+    SANCTIONS_SCREEN = "PEPS_SCREEN"
+    SANCTIONS_SCREEN = "SANCTIONS_SCREEN"
```
Comment (hmac) [inline lib/passfort.rb]: Duplicates here

File: lib/passfort.rb [hunk 3]
```diff
@@ -8,6 +8,21 @@
...
+    ADVERSE_MEDIA_SCREEN = "PEPS_AND_SANCTIONS_SCREEN"
```
Comment (hmac) [inline lib/passfort.rb]: these are different

File: lib/passfort.rb [hunk 4]
```diff
@@ -8,6 +8,21 @@
...
+    ADVERSE_MEDIA_SCREEN = "ADVERSE_MEDIA_SCREEN"
+    COMPANY_OWNERSHIP = "COMPANY_REGISTRY"
```
Comment (hmac) [inline lib/passfort.rb]: and here

File: lib/passfort.rb [hunk 5]
```diff
@@ -8,6 +8,21 @@
...
+    COMPANY_OWNERSHIP = "COMPANY_OWNERSHIP"
+    COMPANY_FILING_PURCHASE = "COMPANY_FILINGS"
```
Comment (hmac) [inline lib/passfort.rb]: and here

File: lib/passfort.rb [hunk 6]
```diff
@@ -8,6 +8,21 @@
 require "passfort/client"
 
 module Passfort
+  module CheckType
+    IDENTITY_CHECK = "IDENTITY_CHECK"
+    DOCUMENT_VERIFICATION = "DOCUMENT_VERIFICATION"
+    DOCUMENT_FETCH = "DOCUMENT_FETCH"
+    PEPS_SCREEN = "PEPS_SCREEN"
+    SANCTIONS_SCREEN = "SANCTIONS_SCREEN"
```
Comment (hmac) [inline lib/passfort.rb]: I couldn't find a reference to `PEPS_SCREEN` or `SANCTIONS_SCREEN` in the docs - are you sure they exist?

File: lib/passfort.rb [hunk 7]
```diff
@@ -8,6 +8,21 @@
...
+    PEPS_AND_SANCTIONS_SCREEN = "PEPS_AND_SANCTIONS_SCREEN"
+    ADVERSE_MEDIA_SCREEN = "ADVERSE_MEDIA_SCREEN"
```
Comment (hmac) [inline lib/passfort.rb]: Couldn't find this check type in the docs either

File: lib/passfort.rb (hunk 6 above)
Comment (acraileanu) [reply lib/passfort.rb]: https://passfort.com/developer/v4/data-reference/CheckResourcePepsScreen
https://passfort.com/developer/v4/data-reference/CheckResourceSanctionsScreen

File: lib/passfort.rb (hunk 7 above)
Comment (acraileanu) [reply lib/passfort.rb]: This one appears in their check types list, but it is not being used. I'll remove it

File: lib/passfort.rb (hunk 6 above)
Comment (hmac) [reply lib/passfort.rb]: Those are `CheckResources`, not check types - can you find a corresponding check type for those? Because I can't.

**Comment by acraileanu** [comment]
I've removed the check that was not being used. The other two are referenced. Will merge it into one single push if all else looks good.

File: lib/passfort.rb (hunk 6 above)
Comment (acraileanu) [reply lib/passfort.rb]: API Reference: https://passfort.com/developer/v4/data-reference/CheckType
Though, as tested, not all seem to work.

**Pull request merged**